tasks/main.yml

```diff
@@ -81,3 +82,10 @@
     owner: '{{ deploy_user }}'
     group: '{{ deploy_group }}'
   when: env == 'production' or env == 'staging'
+
+
```

Comment:
I added that for logical separation. The three tasks above are only for production. The one underneath is for all environments.